Plugins/BridgeJS/Tests/BridgeJSToolTests/__Snapshots__/BridgeJSLinkTests/EnumCase.Export.d.ts

```diff
@@ -0,0 +1,44 @@
+// NOTICE: This is auto-generated code by BridgeJS from JavaScriptKit,
+// DO NOT EDIT.
+//
+// To update this file, just rebuild your project or run
+// `swift package bridge-js`.
+
+export const Direction: {
```

Comment:
I think it makes sense to declare constant enum type definitions at the top-level scope instead of exporting them from `Exports` since they don't require access to wasm instance.

I think we have a few things to fix here:
1. Generate `export const` definition in the corresponding `.js` file as well as TS and bundlers expect it.
2. Remove the enum definition from exports of `createInstantiator` result in `.js` side

Reply:
Thanks for feedback, this definitely makes more sense, addressed this issue and added relevant runtime tests for all types of added enums, will remember about those for future functionalities 🙇🏼‍♂️